tests/estimators/classification/test_deeplearning_common.py

```diff
@@ -509,3 +509,94 @@ def test_learning_phase(art_warning, image_dl_estimator):
         assert hasattr(classifier, "_learning_phase")
     except ARTTestException as e:
         art_warning(e)
+
+
+# Note, the following part is for testing the amp tool on pytorch platform only.
+apex_spec = importlib.util.find_spec("apex")
+if apex_spec is not None:
+    amp_spec = importlib.util.find_spec("apex.amp")
+else:
+    amp_spec = None
+amp_found = amp_spec is not None
+
+
+@pytest.mark.skipif(not amp_found, reason="Skip unittests if apex module is not found.")
+@pytest.mark.only_with_platform("pytorch")
```

Comment:
Will this test a) only ever be able to run on Pytorch only due to some Pytorch dependency, or b) is it that it only passes for Pytorch for now but could eventually pass for other frameworks? 
If case a): then `@pytest.mark.skipMlFramework("tensorflow", "keras", etc..... "non_dl_frameworks")` should be used instead of `@pytest.mark.only_with_platform("pytorch")` (`only_with_platform` is only currently used temporarily while the tests are being refactored)
if case b) then this test should be placed within `test_deeplearning_specific.py` instead of `test_deeplearning_common.py`

Reply:
Solved.